TaskLayer/MetaMorpheusTask.cs

```diff
@@ -301,7 +301,11 @@ protected static List<Protein> LoadProteinDb(string fileName, bool generateTarge
             List<string> dbErrors = new List<string>();
             List<Protein> proteinList = new List<Protein>();
 
-            if (Path.GetExtension(fileName).Equals(".fasta") || Path.GetExtension(fileName).Equals(".fa"))
+            var theExtension = Path.GetExtension(fileName).ToLowerInvariant();
+            bool compressed = theExtension.EndsWith("gz"); // allows for .bgz and .tgz, too which are used on occasion
```

Comment:
can you do a search to check that MetaMorpheus can handle .fa.gz and .fasta.gz? and preferably add that to a unit test